health/tests/health_check.rs

```diff
@@ -10,14 +10,42 @@ use std::time::Duration;
 
 const TEST_SERVICE: &str = "grpc.test.TestService";
 
+#[cfg(feature = "protobuf-codec")]
+mod constants {
+    use crate::proto::ServingStatus;
```

Comment:
The code of health module are generated by xtask, maybe you can update xtask instead. And `Serving` is more rusty than `SERVING`.